docs/spark/how-to-guides/deploy-worker-udf-binaries.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Deploy .NET for Apache Spark worker and user-defined function binaries
+description: Learn how to deploy .NET for Apache Spark worker and user-defined function binaries.
+ms.date: 12/30/2019
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Deploy .NET for Apache Spark worker and user-defined function binaries
+
+This how-to provides general instructions on how to deploy .NET for Apache Spark worker and user-defined function binaries, including which Environment Variables to set up and some commonly used parameters when launching applications with `spark-submit`.
+
+## Configurations
+Configurations show the general environment variables and parameters settings in order to deploy .NET for Apache Spark worker and user-defined function binaries.
+
+### Environment variables
+When deploying workers and writing UDFs, there are a few commonly used environment variables that you may need to set: 
+
+| Environment Variable         | Description
+| :--------------------------- | :---------- 
+| DOTNET_WORKER_DIR            | Path where the <code>Microsoft.Spark.Worker</code> binary has been generated.</br>It's used by the Spark driver and will be passed to Spark executors. If this variable is not set up, the Spark executors will search the path specified in the <code>PATH</code> environment variable.</br>_e.g. "C:\bin\Microsoft.Spark.Worker"_
+| DOTNET_ASSEMBLY_SEARCH_PATHS | Comma-separated paths where <code>Microsoft.Spark.Worker</code> will load assemblies.</br>Note that if a path starts with ".", the working directory will be prepended. If in **yarn mode**, "." would represent the container's working directory.</br>_e.g. "C:\Users\&lt;user name&gt;\&lt;mysparkapp&gt;\bin\Debug\&lt;dotnet version&gt;"_
```

Comment:
I don't think this renders properly. 

![image](https://user-images.githubusercontent.com/38858737/72825924-26328600-3c35-11ea-9600-e15ae054b677.png)

Reply:
Changed. Thanks!